src/renderer/components/Header/AppInfo.tsx

```diff
@@ -14,21 +47,185 @@ export const AppInfo = memo(() => {
+        <>
+            <Box w="full">
+                <HStack
+                    ml={0}
+                    mr="auto"
+                >
+                    <Image
+                        boxSize="36px"
+                        draggable={false}
+                        src={logo}
+                    />
+                    <Heading size="md">chaiNNer</Heading>
+                    <Tag>Alpha</Tag>
+                    <Tag>{`v${appVersion ?? '#.#.#'}`}</Tag>
+                    {updateVersion && (
+                        <Tooltip
+                            closeOnClick
+                            closeOnMouseDown
+                            borderRadius={8}
+                            label={`Update available (${updateVersion.tag_name})`}
+                            px={2}
+                            py={1}
+                        >
+                            <IconButton
+                                aria-label="Update available"
+                                colorScheme="green"
+                                icon={<DownloadIcon />}
+                                variant="ghost"
+                                onClick={onModalOpen}
+                            />
+                        </Tooltip>
+                    )}
+                </HStack>
+            </Box>
+            <Modal
+                isCentered
+                isOpen={isModalOpen}
+                scrollBehavior="inside"
+                size="xl"
+                onClose={onModalClose}
+            >
+                <ModalOverlay />
+                <ModalContent
+                    bgColor="var(--chain-editor-bg)"
+                    h="calc(100% - 7.5rem)"
+                    maxW="unset"
+                    my={0}
+                    overflow="hidden"
+                    w="calc(100% - 7.5rem)"
+                >
+                    <ModalHeader>Update Changelog</ModalHeader>
+                    <ModalCloseButton />
+                    <ModalBody bgColor="var(--bg-900)">
+                        <ReactMarkdown
+                            skipHtml
+                            components={ChakraUIRenderer()}
+                        >
+                            {changelog ?? ''}
+                        </ReactMarkdown>
+                    </ModalBody>
+
+                    <ModalFooter>
+                        <HStack>
+                            <Button
+                                variant="ghost"
+                                onClick={onModalClose}
+                            >
+                                Close
+                            </Button>
+                            <Button
+                                isExternal
+                                as={Link}
+                                colorScheme="green"
+                                href={updateVersion?.html_url}
+                            >
+                                Download from GitHub
+                            </Button>
```

Comment:
I don't think we need the GH download link. Our download page is more user-friendly anyway.

Reply:
I agree that our page is more user friendly but I'd still like to link this. I see no reason not to

Reply:
Why do want to link it? Or phrased differently: what additional value does it provide?

I just see it as an option that could confuse new users. We want people to update, so I think that we should make that process simple. This is why I like the website: it auto selects the correct OS. I just don't think that new users will have an easy time picking the correct download from our GH releases.

As for advanced users: they should be able to navigate to the GH releases themselves. If you think that's too many clicks, then we could add a "View this release on GitHub" link to the download page.